src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -852,7 +852,7 @@ async Task<(bool, ImmutableArray<ProjectConfiguration>)> CheckAsync(UpToDateChec
 
                 globalProperties.TryGetValue(FastUpToDateCheckIgnoresKindsGlobalPropertyName, out string? ignoreKindsString);
 
-                (LogLevel requestedLogLevel, Guid projectGuid) = await WhenAll(
+                (LogLevel requestedLogLevel, Guid projectGuid) = await(
```

Comment:
Nit: You may want to adjust the spacing between `await` and `(` to emphasize that this is awaiting a tuple, rather than calling a method named await.

Reply:
```suggestion
                (LogLevel requestedLogLevel, Guid projectGuid) = await (
```